### PR TITLE
Use board

### PR DESCRIPTION
**Pull request by DHervochon**
Important bug fixes to the board.
Main now uses the board object as well as creating it.
Print methods created for board.
method for getting an astar friendly pathable array out of the board now exists and _is used_ for getting our path.
Some stuff to make the linter shout less at me/us.
Tested it this time.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@
 *.pyc
 battlesnake
 .idea
-.vscode
+.vscode
```
Comment (DHervochon) [inline .gitignore:5]: just ignore this one I changed my mind about something and couldn't figure out how to cleanly remove it from history

File: app/board.py [hunk 2]
```diff
@@ -7,20 +7,25 @@ class Board:
...
                     is_food = True
 
                 new_tile = t.Tile(x, y, path_board[y][x], is_food, char_board[y][x])
-                self._the_board[y].append(new_tile)
+                new_row.append(new_tile)
+                
                 if new_tile.get_food():
                     self.food_tiles.append(new_tile)
+            
+            self._the_board.append(new_row)
```
Comment (DHervochon) [inline app/board.py]: I don't know why it didn't work before but I know it works now. ¯\_(ツ)_/¯

Comment (DHervochon) [reply app/board.py]: figured it out in the shower, I'm a dumbass. The original line 21 needed to be:
self._the_board[y][x] = new_tile
new way is fine and tested so idk if it's worth fixing the original instead.

Comment (kylebisley) [reply app/board.py]: This is difficult for me to read.  Please add comments explaining the general steps.

**Review by kylebisley — CHANGES_REQUESTED** [review]

**Review by kylebisley — APPROVED** [review]

**Pull request merged**